rho/facts.py

```diff
@@ -185,6 +188,8 @@ def new_fact(fact_name, description, is_default=None,
 new_fact('jboss.fuse.activemq-ver', 'ActiveMQ version', is_default=False)
 new_fact('jboss.fuse.camel-ver', 'Camel version', is_default=False)
 new_fact('jboss.fuse.cxf-ver', 'CXF version', is_default=False)
+new_fact('jboss.fuse.fuse-on-eap', 'Fuse on EAP',
+         is_default=True, categories=[JBOSS_FACTS])
```

Comment:
Brainstorm related to above comment. Maybe if there was some python binding where we could import this file and call a function like `facts.list.('jboss.fuse')` it would return all the facts under 'jboss.fuse'.

Reply:
You could potentially import `facts.ALL_FACTS`, which is a set holding the names of all facts in Rho, and then filter it for ones that start with `jboss.fuse` or whatever string you want.

At some point I want to come up with a more stable API for getting a list of facts, but we're not there yet.